docs/learning/ui/framework/Frontstages.md

```diff
@@ -8,7 +8,191 @@ A **Frontstage** is a full-screen configuration designed to enable the user to a
 |**Nested** | is accessed from a primary frontstage. It may use all zones and panels, but instead of the App button, the Tool Widget contains a Back button to return to the primary frontstage.
 |**Modal** | is accessed from another frontstage or the Backstage. It may contain any content along with a Back button. It does not use zones or stage panels. It is useful for application settings and data management user interfaces.
 
-## Configuring a Frontstage
+## Frontstage shown using UI 2.0 components
```

Comment:
Do you want to refer to it at AppUI instead of UI 2.0 here? Is there a difference?

Reply:
Updated to hopefully make in better.

Reply:
Sounds better now. I'd like to take a pass at the whole docs structure to help discoverability but that can definitely be outside of this PR.